File: plugins/majestic-tools/skills/agent-scaffold/resources/python/agent.py
import sys
from claude_agent_sdk import Agent

# Configure allowed tools based on user selection
ALLOWED_TOOLS = {{tools_list}}

agent = Agent(
    model="claude-sonnet-4-20250514",
    allowed_tools=ALLOWED_TOOLS,
    system_prompt="""{{system_prompt}}""",
)


def main():
    prompt = sys.argv[1] if len(sys.argv) > 1 else "Hello, what can you help me with?"
    result = agent.run(prompt=prompt)
    print(result.output)


if __name__ == "__main__":
    main()

File: plugins/majestic-agent-sdk/skills/agent-sdk/resources/typescript/agent.ts
import { Agent } from '@anthropic-ai/claude-agent-sdk';

// Configure allowed tools based on user selection
const allowedTools = {{tools_array}};

const agent = new Agent({
  model: 'claude-sonnet-4-20250514',
  allowedTools,
  systemPrompt: `{{system_prompt}}`,
});

async function main() {
  const result = await agent.run({
    prompt: process.argv[2] || 'Hello, what can you help me with?',
  });

  console.log(result.output);
}

main().catch(console.error);
